policy/release/olm/olm.rego

```diff
@@ -187,8 +187,7 @@ deny contains result if {
 #     Ensure all related images are available. The related images are defined by
 #     an file containing a json array attached to the validated image. The digest
 #     of the attached file is pulled from the RELATED_IMAGES_DIGEST result.
-#   collections:
-#   - redhat
+#
```

Comment:
Consider adding a comment here somehow (not in the metadata) to explain why it's no in the collection.